spec/history_spec.rb

```diff
@@ -22,36 +22,36 @@
   end
 
   describe ".default_file" do
-    let(:xdg_name) { 'XDG_DATA_HOME' }
-    let(:default_path) { File.expand_path '~/.pry_history' }
-
-    def stub_hist(options)
```

Comment:
We probably shouldn't be nuking this method. It was created so we can easily test various scenarios. The missing test needs this scenario:

```rb
stub_hist has_default: true, xdg_home: '/my/path'
```

`has_default` here is the existence of `~/.pryrc` and `xdg_home` is the path to the XDG-compatible config.

Does that make sense?

Reply:
Updated the code.